android/src/main/java/com/adyenreactnativesdk/ui/LoadingDialogFragment.kt

```diff
@@ -0,0 +1,39 @@
+package com.adyenreactnativesdk.ui
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.DialogFragment
+import com.adyenreactnativesdk.R
+
+interface Cancelable {
+    fun canceled()
+}
+
+class LoadingDialogFragment(private val cancelable: Cancelable) : DialogFragment() {
```

Comment:
Is the purpose gonna be loading? otherwise naming is confusing 😄

Reply:
It is more of "Pending payment", indeed 🤔